pozo/graphs.py

```diff
@@ -122,6 +131,52 @@ def add_las_object(self, ar, **kwargs):
             self.add_tracks(data)
         if include and len(include) != 0:
             self.reorder_all_tracks(include)
+ 
+    def add_welly_object(self, ar, **kwargs):
+        include = kwargs.get('include', [])
+        exclude = kwargs.get('exclude', [])
+        yaxis = kwargs.get('yaxis', None)
+        yaxis_name = kwargs.get('yaxis_name',"DEPTH")
+        yaxis_unit = kwargs.get('yaxis_unit', None)
+        
+        warnings.filterwarnings("ignore", message="Index.is_numeric is deprecated.")
+        for curve in ar.data.values():
+            print(f"{curve.mnemonic} w/ units: {curve.units} w/ shape {curve.values.shape}")
```

Comment:
ya no

Reply:
Are you talking about the filterwarnings?

Reply:
Done